src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1029,6 +1029,29 @@ public function upsert(array $values, $uniqueBy, $update = null)
         );
     }
 
+    /**
+     * Update the column's update timestamp.
+     *
+     * @param  string|null  $column
+     * @return int|false
+     */
+    public function touch($column = null)
+    {
+        $time = $this->model->freshTimestamp();
+
+        if ($column) {
+            return $this->toBase()->update([$column => $time]);
+        }
+
+        $column = $this->model->getUpdatedAtColumn();
+
+        if (! $this->model->usesTimestamps() || is_null($column)) {
+            return false;
+        }
+
+        return $this->toBase()->update([$column => $time]);
```

Comment:
```suggestion
        $time = $this->model->freshTimestamp();
        $column = $column ?: ($this->model->usesTimestamps() ? $this->model->getUpdatedAtColumn() : null);

        if (is_null($column) {
            return false;
        }

        return $this->toBase()->update([$column => $time]);
```
How about using this instead then (or a variation of this without the ternaries if you'd prefer)? That way we can avoid the duplicated calls / early returns.

Reply:
I think it's fine the way it is. I want the structure to be as close to `Model::touch()` as possible.

Appreciate the PR review 👍

Reply:
No problem, I didn't check Model::touch() so if it's matching the existing structure it should be fine.